### ISSUE TITLE
Black Screen when in Window w/ Other Controls

### ISSUE THREAD

**Issue by davecom** [slycrel/JSTileMap#19]
I'm not yet sure if this a JSTileMap issue, SpriteKit bug, or mistake on my part. I had a tile set working well in a SKView that took up the entirety of an OS X window.  I now added controls to the top part of the window and constrained the SKView (where the tile map is loaded) to the bottom of the window.  I kept the size of the SKView the same.  However, without any code changes (just changes in Interface Builder) the SKView now just draws a big black box.  Has anyone seen behavior like this?

![screen shot 2014-10-08 at 2 15 49 am](https://cloud.githubusercontent.com/assets/424688/4554725/5d71eb80-4eb3-11e4-9860-c7a0a280db7d.png)

The code/.xib doing this is here:
https://github.com/davecom/Crump

**Comment by slycrel** [comment]
hi @davecom 
I never did find any issues with your code, and am still unsure why you were seeing a blank screen.  I checked again today and after updating your project to modern swift, and commenting out your pseudo-dynamic enemy placement code, I was able to run this again just fine.  (I do not know why swift thinks that the object group dictionaries are SKNodes)
I hope you got past this specific issue when you were working on it before.
